### DISCUSSION TITLE
Osu mania keys doesn't hold!

### DISCUSSION THREAD

**Discussion by Flex146** [ppy/osu#19650] [Q&A] [answered]
### Type

Game behaviour

### Bug description

I wanted to play osu mania so I connected my bluetooth keyboard to my phone, but when I was playing I couldn't hit the hold notes, it just pressed one time and then it releases instantly when I was still holding the key, then it starts pressing fast like if I was spamming the key. 

### Version

2022.731.1

### Logs

.

**Comment by peppy** [comment, marked as answer]
This is probably the way that your bluetooth keyboard operates.

**Reply by Flex146** [reply]
isn't there a way to change it?

**Reply by peppy** [reply]
No idea. Please use google. It's definitely not something our end.

**Reply by Flex146** [reply]
oh, thanks then
